### PR TITLE
[Android] Support Action.isEnabled

### LINKED ISSUE #5719: [Android] Action isEnabled

### PR DESCRIPTION
**Pull request by golddove**
# Related Issue

Fixes #5719 

# Description

- Re-SWIG'd shared model changes.
- Elements with `selectAction` or `inlineAction` with `isEnabled: false` are not clickable.
- RichTextBlock's Inline with `selectAction` with `isEnabled: false` are rendered as if there is no `selectAction` provided
- Top-level actions and `ActionSet actions with `isEnabled: false` are not clickable AND get platform's disabled button appearance.
- Custom actions are expected to determine how to support `isEnabled` (the SDK doesn't impose any enabling/disabling)

# Sample Card

Updated `v1.5\Elements\Action.isEnabled` sample to include RichTextBlock

# How Verified

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5776)

### CODE REVIEW

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java [hunk 1]
```diff
@@ -216,7 +217,7 @@ public static void setBackgroundImage(RenderedAdaptiveCard renderedCard,
 
     public static void setSelectAction(RenderedAdaptiveCard renderedCard, BaseActionElement selectAction, View view, ICardActionHandler cardActionHandler, RenderArgs renderArgs)
     {
-        if (selectAction != null)
+        if (selectAction != null && selectAction.GetIsEnabled())
         {
             view.setFocusable(true);
```
Comment (jwoo-msft) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java:222]: nit: should it be focusable? will Android VO be able to narrate the disabled element?

**Review by jwoo-msft — COMMENTED** [review]
looks great; except for the one question, I had in the review.

File: samples/v1.5/Elements/Action.IsEnabled.json [hunk 2]
```diff
@@ -21,7 +21,28 @@
         "title": "Disabled",
         "isEnabled": false
       }
-    }
+    },
+		{
```
Comment (paulcam206) [inline samples/v1.5/Elements/Action.IsEnabled.json:25]: indenting looks off here

**Review by paulcam206 — APPROVED** [review]

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java (hunk 1 above)
Comment (golddove) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java:222]: No, TalkBack will not narrate the element. 

From the spec (#4670):

> Images or other elements with a disabled selectAction look and behave as if they had no selectAction.

**Comment by golddove** [comment]
> have you gotten VSCode set up with the [editorconfig extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)?
> 
> also, something something emacs

Yes! Yet, it thinks this file should have tabs (bottom info bar).

**Comment by paulcam206** [comment]
> > have you gotten VSCode set up with the [editorconfig extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)?
> 
> Yes! Yet, it thinks this file should have tabs (bottom info bar).

Actually,
https://github.com/microsoft/AdaptiveCards/blob/a05ba759ac15ce08c503650ef3edea8fa9e4a24d/.editorconfig#L10-L11
 
> > also, something something emacs
> 
> I'd sooner use vim :)

you can do that in emacs!

**Review by jwoo-msft — APPROVED** [review]
looks great; except for the one question, I had in the review.

**Pull request merged**